engine.ini

```diff
@@ -0,0 +1,11 @@
+[benchmark]
+[core]
+debug = false
+observation_workers = 2
+reset_retries = 1
+[controllers]
```

Comment:
This second config file is used just for testing out separate configuration and will be removed.